roles/jenkins_slave/tasks/main.yml

```diff
@@ -0,0 +1,77 @@
+- name: install rpm packages
+  yum:
+    name: "{{ jslave_rpm_deps | join(',') }}"
+    state: present
+
+- name: install pip packages
```

Comment:
It would be nice if we could eventually move this to not be using pip to install globally. That's generally a big no-no, when Python is handled by an external package manager like ours is. Then you have pip and yum competing over certain files.

Obviously this is how it's already done, but it's something for us to consider going forward.